### PR TITLE
fixes for R1 mixer https://github.com/PX4/sitl_gazebo/issues/533

### PR DESCRIPTION
**Pull request by bperseghetti**
Fix for mixer issue observed with R1 in Gazebo.
https://github.com/PX4/sitl_gazebo/issues/533

### CODE REVIEW

**Comment by julianoes** [comment]
@bperseghetti thanks for the contribution!

**Comment by bperseghetti** [comment]
> Giving it a second look, I don't think this is the mixer file being used in the aion robotics r1 rover
> 
> The mixers are specified in
> 
> https://github.com/PX4/Firmware/blob/120f188adabae13f8d7b00a8fbd5c2db7f01f046/ROMFS/px4fmu_common/init.d-posix/1061_r1_rover#L38
> 
> which points to https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/mixers/generic_diff_rover.main.mix
> I believe this commit will break the default rover model which is using this [mixer file.](https://github.com/PX4/Firmware/blob/120f188adabae13f8d7b00a8fbd5c2db7f01f046/ROMFS/px4fmu_common/init.d-posix/1060_rover#L37)
> 
> Am I missing something? Did this change actually result in a change of behavior in SITL?

As I mentioned before that is not the file the r1 sim actually uses as you can see from my image.

image

**Comment by bperseghetti** [comment]
Good point on the regular non-skid-steer rover though.

**Comment by bperseghetti** [comment]
@Jaeyoung-Lim @julianoes Wow, interesting. Looks like it finds 1061 but it loads the 1061_tf-r1 even though we are providing it with the vehicle name "r1_rover" and should be using the 1061_r1_rover.

**Comment by bperseghetti** [comment]
@TSC21 @Jaeyoung-Lim I see you fixed the the one issue I pointed out. However we still need to add the mixer to the sitl side. Please review these changes.

**Comment by bperseghetti** [comment]
@Jaeyoung-Lim sure. One sec.

File: ROMFS/px4fmu_common/mixers-sitl/rover_ackermann_sitl.main.mix [hunk 1]
```diff
@@ -43,4 +43,4 @@ Z:
 
 Output 8
 ---------------------------------------
-Z:
+Z:
```
Comment (Jaeyoung-Lim) [inline ROMFS/px4fmu_common/mixers-sitl/rover_ackermann_sitl.main.mix:46]: Missing EOL

Comment (bperseghetti) [reply ROMFS/px4fmu_common/mixers-sitl/rover_ackermann_sitl.main.mix:46]: odd, I just did an mv for that one. Fixed anyhow. Give me a sec to add the other file back and commit.

**Comment by bperseghetti** [comment]
Guess I need one more EOL one sec

**Comment by bperseghetti** [comment]
Building and testing now.

**Comment by bperseghetti** [comment]
image
All works correctly now.

**Pull request merged**